.yarnrc.yml

```diff
@@ -1,11 +1,36 @@
-compressionLevel: mixed
+compressionLevel: 0
 
-enableGlobalCache: false
+enableGlobalCache: true
 
 installStatePath: ./.yarn/root-install-state.gz
 
+logFilters:
+  - code: YN0007
+    level: discard
+  - code: YN0005
+    level: discard
+  - code: YN0076
+    level: discard
+
 nodeLinker: node-modules
 
 npmPublishAccess: public
 
+tsEnableAutoTypes: true
+
+npmRegistryServer: "https://registry.yarnpkg.com"
+
+supportedArchitectures:
+  cpu:
+    - current
+    - x64
+    - arm64
+  os:
+    - current
```

Comment:
What does current mean here?

Reply:
This confuses me
```
    - current
    - linux
    - darwin
```

To me this implies:
> Can you use `linux` or `osx` or whatever you are currently using.

Is that incorrect?

If the "current" os is always supported, then the mention of the 2 specific os/cpu doesn't add anything, and the whole supportedArchitecture field is not useful to specify.